contracts/token/Cargo.toml

```diff
@@ -8,8 +8,22 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-soroban-sdk = { workspace = true }
-soroban-token-sdk = { workspace = true }
+soroban-sdk = { version = "21.6.0" }
+soroban-token-sdk = { version = "21.6.0" }
 
 [dev-dependencies]
-soroban-sdk = { workspace = true, features = ["testutils"] }
+soroban-sdk = { version = "21.6.0", features = ["testutils"] }
```

Comment:
Why so?

Reply:
huh, strange, I seem to have discarded the wrong toml. Fixed now